koku/masu/processor/parquet/ocp_cloud_parquet_report_processor.py

```diff
@@ -151,8 +151,9 @@ def process(self, parquet_base_filename, daily_data_frames):
             matched_tags = []
             if self.has_enabled_ocp_labels:
                 LOG.info("Getting matching tags from Postgres.")
+                matching_enabled_tags = self.db_accessor.check_for_matching_enabled_keys()
                 matched_tags = self.db_accessor.get_openshift_on_cloud_matched_tags(self.bill_id, report_period_id)
-                if not matched_tags:
+                if not matched_tags and matching_enabled_tags:
```

Comment:
Maybe I am mis-understanding the purpose of this PR, but aren't we trying to prevent the code in `get_openshift_on_cloud_matched_tags` from running if there are no enabled tags?

Reply:
We are trying to prevent the `get_openshift_on_cloud_matched_tags_trino` from running, but after thinking about it we can also prevent the `get_openshift_on_cloud_matched_tags` from running as well.